source/lex.tex

```diff
@@ -1352,8 +1352,8 @@
 alert           &   BEL             &   \tcode{\textbackslash a}                \\
 backslash       &   \textbackslash  &   \tcode{\textbackslash\textbackslash}    \\
 question mark   &   ?               &   \tcode{\textbackslash ?}                \\
-single quote    &   \tcode{'}       &   \tcode{\textbackslash\tcode{'}}         \\
-double quote    &   \tcode{"}       &   \tcode{\textbackslash\tcode{"}}         \\
+single quote    &   \tcode{'}       &   \tcode{\textbackslash '}                \\
+double quote    &   \tcode{"}       &   \tcode{\textbackslash "}                \\
```

Comment:
I omitted this one because it should make no difference to the rendered output.

Reply:
Agreed, see above, removing.